### DISCUSSION TITLE
Nmblookup

### DISCUSSION THREAD

**Discussion by frasou** [christgau/wsdd#88] [Q&A] [answered]
Hello,
without wsdd nmblookup does't work. When I ran wsdd in discovery mode, nmblookup was able to find win10 peers. 
I am just curious how does this work? Is wsdd using some sort of database which nmblookup reads a list of peers from?

**Comment by christgau** [comment, marked as answer]
No, wsdd is not feeding some kind of database and I am not aware of any interaction between nmblookup and wsdd. At least there is no intended one and technically they both differ quite a lot.

**Reply by frasou** [reply]
Thank you for your answer. So it must be some kind of magic. Nmblookup can't find any host/IP without wssd running. Interesting.

**Reply by christgau** [reply]
Really sounds interesting. Although the discussion/question is marked as answered, would you like to share some details about your setup, like which hosts are involved and which software is running on them?

**Reply by frasou** [reply]
I have a small company, there is a one linux box and three WIN10 HOME edition client computers. We have a group named WORKGROUP.

Without wsdd the Win10 clients are not able to see samba server on the linux box. And vice versa, linux is not able to see any of win10 machines. By IP I am able to connect to those machines without a problem. But network discovery or nmblookup doesn't work.

When wsdd is running, everything works perfectly. Even nmblookup. Strange.
